frontend/src/components/Services/ServicesRow/index.jsx

```diff
@@ -0,0 +1,41 @@
+import style from './ServicesRow.module.scss';
+import PropTypes from 'prop-types';
+import Button from '../../lib/Button';
+import { minToDuration } from '@/utils/formater/time';
+
+function ServicesRow({
+  name,
+  description,
+  price,
+  duration,
+}) {
+  return (
+    <div className={style.ServicesRow}>
+      <div className={style.ServicesRowDesc}>
+        <span className={style.ServicesRowDescName}>{name}</span>
+        <div className={style.ServicesRowDescDetail}>
+          <span>
+            {description}
+          </span>
+        </div>
+      </div>
+      <div className={style.ServicesRowInfos}>
+        <span>{minToDuration(duration)}</span>
+        <span>•</span>
+        <span>{price} €</span>
+      </div>
+      <Button className={style.ServicesRowBtn} variant="dark">Choisir</Button>
```

Comment:
Une trad ici ?

Et si tu veux mettre du style dessus, wrap le dans un composant. La tu écrase le style du composant

```suggestion
     <div className={style.ServicesRowBtn}>
        <Button variant="dark">Choisir</Button>
      </div>
```

```scss
justify-self: flex-end;
```